src/common/utils/Types.h

```diff
@@ -12,12 +12,13 @@
 namespace nebula {
 
 enum class NebulaKeyType : uint32_t {
-  kVertex = 0x00000001,
+  kTag_ = 0x00000001,
```

Comment:
`kTag`?

Reply:
`kTag` is defined in Base.h `constexpr char kTag[] = "_tag";`.And compiler will report a warning if a local variable overwrite global variables.